### ISSUE TITLE
Light theme support

### ISSUE THREAD

**Issue by zzhaolei** [kpitt/zed-theme-intellij-newui#2]
Are there any plans to support light theme?

**Comment by kpitt** [comment]
Yes, my intention was to incorporate support for the light theme also, but it's a lower priority because I don't actually use light mode myself. It's on hold right now because I've been too busy with other projects to commit any time to it, but hopefully it will still happen eventually.

**Comment by unorsk** [comment]
Looking forward to the Light theme!
